tests/UnitTests/Cogworks.AzureSearch.UnitTests/Operations/IndexOperationTests.cs

```diff
@@ -154,6 +154,27 @@ public async Task Should_CreateOrUpdateIndex()
             Assert.Equal($"Index {TestDocumentModelDefinition.IndexName} successfully created or updated.", operationResult.Message);
         }
 
+        [Fact]
+        public async Task Should_CreateOrUpdateCustomIndex()
+        {
+            // Arrange
+            var createdOrUpdatedIndex = new Index
+            {
+                Name = TestDocumentModelDefinition.IndexName
+            };
+
+            _ = IndexOperationWrapper.CreateOrUpdateAsync<TestDocumentModel>(Arg.Any<Index>(), Arg.Any<bool>())
```

Comment:
```_ = IndexOperationWrapper.CreateOrUpdateAsync<TestDocumentModel>(createdOrUpdatedIndex, Arg.Any<bool>())```

Reply:
depends on bool flag if just text or index. Or we should simplify it and pass always object 🤔What do you think ?